core/src/main/java/fr/sncf/osrd/standalone_sim/ScheduleMetadataExtractor.kt

```diff
@@ -40,6 +40,8 @@ import mu.KotlinLogging
 
 private val logger = KotlinLogging.logger {}
 
+const val CLOSED_SIGNAL_RESERVATION_MARGIN = 20.0
```

Comment:
Could you add some documentation on that constant?

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/9195/commits/d4e146bff6000da1b381f62fefdb24c80a92f703